config.xmi

```diff
@@ -948,6 +953,8 @@
   </styleConfig>
   <styleConfig name="PS_ProtectedSite">
```

Comment:
This styleConfig element is now missing a closing tag if I do not miss anything.